petstore/frontend/react/src/about/About.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
All the files named "index.js" were renamed so that when we are editing multiple of them, we don't see a bunch of editor tabs that all just say "index.js". I find that hard to work with.